<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>661</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabUser">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Objednávky:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxCar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Úprava záznamu o aute: </string>
      </property>
      <widget class="QLineEdit" name="lineEditCarBrand">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCarOK">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCarModel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCarYear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxService">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Objednanie služby: </string>
      </property>
      <widget class="QComboBox" name="comboBoxService">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAddService">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Pridaj objednávku</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxFaktura">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Faktúra: </string>
      </property>
      <widget class="QPushButton" name="pushButtonFaktura">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Tlač faktúry </string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoveOrder">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zruš objednávku</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetOrders">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Služba</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stav služby</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWorker">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetOrdersWorker">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>301</width>
        <height>261</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>používateľ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>služba</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>stav</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Objednávky: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Správa používateľov: </string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetUsers">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>191</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonLoggOutUser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Odhlásiť sa </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Súbor</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoggOut"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>Používateľ</string>
    </property>
    <addaction name="actionAddUser"/>
    <addaction name="actionEditUser"/>
    <addaction name="actionDeleteUser"/>
   </widget>
   <widget class="QMenu" name="menuServices">
    <property name="title">
     <string>Služby</string>
    </property>
    <addaction name="actionUploadFileServices"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuUser"/>
   <addaction name="menuServices"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Uložiť</string>
   </property>
  </action>
  <action name="actionLoggOut">
   <property name="text">
    <string>Odhlasiť sa</string>
   </property>
  </action>
  <action name="actionAddUser">
   <property name="text">
    <string>Pridaj používateľa</string>
   </property>
  </action>
  <action name="actionEditUser">
   <property name="text">
    <string>Uprav používateľa</string>
   </property>
  </action>
  <action name="actionDeleteUser">
   <property name="text">
    <string>Vymaž používateľa</string>
   </property>
  </action>
  <action name="actionUploadFileServices">
   <property name="text">
    <string>Nahraj súbor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
